devicetypes/smartthings/2015-samsung-smart-tv.src/2015-samsung-smart-tv.groovy

```diff
@@ -0,0 +1,140 @@
+/**
+ *  2015 Samsung Smart TV
+ *
+ *  Copyright 2015 Min-kyu Park
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition (name: "2015 Samsung Smart TV", namespace: "smartthings", author: "Min-kyu Park") {
+	 	capability "Switch"
+		capability "Button"
```

Comment:
Generally capability "Button" is used for individual button devices that might trigger an action or event. It isn't usually used for device tiles. If the TV can also accept a specific volume level as a percentage it would be advisable to use the "Switch Level" capability. This will allow you to do things like "When my Sleep Sense detects that I am in bed, turn the TV volume down to 15%".